### PR TITLE
Remove asset detail popup in asset configure's camera feed

### LINKED ISSUE #8844: Remove the camera name and details pop-up from the camera feed component on the asset configuration page.
**Describe the bug**

When we are on the asset configuration page, the camera feed component shows the camera name and details pop-up, which is not relevant to the page.

**To Reproduce**

1. Go to any camera asset in the assets page
2. Click on configuration
3. See error

**Expected behavior**

Remove the camera name and details pop-up from the camera feed component on the asset configuration page.

note: This behavior change should only affect the asset configuration page.

**Screenshots**

Image

### PR DESCRIPTION
**Pull request by Rishith25**
## Proposed Changes

- Fixes #8844 
- Removal of the camera name and details pop-up from the camera feed component on the asset configuration page
- Ensured that this change does not affect the functionality of the camera feed component on other pages.

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
LGTM

File: src/Components/CameraFeed/CameraFeed.tsx [hunk 1]
```diff
@@ -28,6 +28,8 @@ interface Props {
   shortcutsDisabled?: boolean;
   onMove?: () => void;
   operate: ReturnType<typeof useOperateCamera>["operate"];
+  //Assest Configuration
+  isAssetConfigurationPage?: boolean;
```
Comment (khavinshankar) [inline src/Components/CameraFeed/CameraFeed.tsx]: This prop seems very rigid, make it a bit generalized, something like showMonitorAsset or hideMonitorAsset.

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/Components/CameraFeed/CameraFeed.tsx (hunk 1 above)
Comment (Rishith25) [reply src/Components/CameraFeed/CameraFeed.tsx]: I have done the requested changes but now some tests are failing.

File: src/Components/CameraFeed/CameraFeed.tsx [hunk 2]
```diff
@@ -145,6 +147,8 @@ export default function CameraFeed(props: Props) {
     />
   );
 
+  const hideMonitorAsset = props.hideMonitorAsset || false;
```
Comment (rithviknishad) [inline src/Components/CameraFeed/CameraFeed.tsx]: not needed

```suggestion
```

File: src/Components/CameraFeed/CameraFeed.tsx [hunk 3]
```diff
@@ -183,15 +187,18 @@ export default function CameraFeed(props: Props) {
...
-            <div className="flex items-center gap-2">
-              <span className="text-xs font-bold md:text-sm">
-                {props.asset.name}
-              </span>
-              <MonitorAssetPopover
-                asset={props.asset}
-                className="absolute z-[100] mt-2 w-56 -translate-x-full -translate-y-4 rounded-md bg-white md:w-[350px] md:-translate-x-full md:-translate-y-2"
-              />
-            </div>
+            {!hideMonitorAsset && (
```
Comment (rithviknishad) [inline src/Components/CameraFeed/CameraFeed.tsx]:
```suggestion
            {!props.hideAssetInfo && (
```

File: src/Components/CameraFeed/CameraFeed.tsx [hunk 4]
```diff
@@ -28,6 +28,8 @@ interface Props {
   shortcutsDisabled?: boolean;
   onMove?: () => void;
   operate: ReturnType<typeof useOperateCamera>["operate"];
+  //Monitor Hide or Show
+  hideMonitorAsset?: boolean;
```
Comment (rithviknishad) [inline src/Components/CameraFeed/CameraFeed.tsx]: 1. don't add comments for obvious things
2. what has the word `monitor` got to do over here? How about making the name more appropriate `hideAssetInfo`?

```suggestion
  hideAssetInfo?: boolean;
```

File: src/Components/CameraFeed/ConfigureCamera.tsx [hunk 5]
```diff
@@ -33,6 +33,8 @@ import CheckBoxFormField from "../Form/FormFields/CheckBoxFormField";
 interface Props {
   asset: AssetData;
   onUpdated: () => void;
+  //show or hide camera pop-up
+  hideMonitorAsset?: boolean;
```
Comment (rithviknishad) [inline src/Components/CameraFeed/ConfigureCamera.tsx]: Why make this a prop? This is always going to be set to true on all it's usages. 

```suggestion
```

File: src/Components/CameraFeed/ConfigureCamera.tsx [hunk 6]
```diff
@@ -58,6 +60,7 @@ export default function ConfigureCamera(props: Props) {
   }>();
   const [presetName, setPresetName] = useState("");
   const [showUnlinkConfirmation, setShowUnlinkConfirmation] = useState(false);
+  const hideMonitorAsset = props.hideMonitorAsset || false;
```
Comment (rithviknishad) [inline src/Components/CameraFeed/ConfigureCamera.tsx]: not needed

```suggestion
```

File: src/Components/CameraFeed/ConfigureCamera.tsx [hunk 7]
```diff
@@ -106,7 +109,12 @@ export default function ConfigureCamera(props: Props) {
   if (!["DistrictAdmin", "StateAdmin"].includes(authUser.user_type)) {
     return (
       <div className="w-full overflow-hidden rounded-lg bg-white shadow">
-        <CameraFeed asset={props.asset} key={key} operate={operate} />
+        <CameraFeed
+          asset={props.asset}
+          key={key}
+          operate={operate}
+          hideMonitorAsset={hideMonitorAsset}
```
Comment (rithviknishad) [inline src/Components/CameraFeed/ConfigureCamera.tsx]:
```suggestion
          hideAssetInfo
```

File: src/Components/CameraFeed/ConfigureCamera.tsx [hunk 8]
```diff
@@ -246,6 +254,7 @@ export default function ConfigureCamera(props: Props) {
                 );
               }
             }}
+            hideMonitorAsset={hideMonitorAsset}
```
Comment (rithviknishad) [inline src/Components/CameraFeed/ConfigureCamera.tsx]:
```suggestion
            hideAssetInfo
```

File: src/Components/Assets/AssetConfigure.tsx [hunk 9]
```diff
@@ -61,7 +61,11 @@ const AssetConfigure = ({ assetId, facilityId }: AssetConfigureProps) => {
       }}
       backUrl={`/facility/${facilityId}/assets/${assetId}`}
     >
-      <ConfigureCamera asset={asset} onUpdated={() => refetch()} />
+      <ConfigureCamera
+        asset={asset}
+        onUpdated={() => refetch()}
+        hideMonitorAsset={true}
+      />
```
Comment (rithviknishad) [inline src/Components/Assets/AssetConfigure.tsx]: can be reverted to how it was before once the above changes are made.
```suggestion
      <ConfigureCamera asset={asset} onUpdated={() => refetch()} />
```

File: src/Components/CameraFeed/CameraFeed.tsx [hunk 10]
```diff
@@ -183,15 +187,18 @@ export default function CameraFeed(props: Props) {
...
-                asset={props.asset}
-                className="absolute z-[100] mt-2 w-56 -translate-x-full -translate-y-4 rounded-md bg-white md:w-[350px] md:-translate-x-full md:-translate-y-2"
-              />
-            </div>
+            {!hideMonitorAsset && (
+              <div className="flex items-center gap-2">
+                <span className="text-xs font-bold md:text-sm">
+                  {props.asset.name}
+                </span>
+                <MonitorAssetPopover
```
Comment (rithviknishad) [inline src/Components/CameraFeed/CameraFeed.tsx]: Feel free to rename the `MonitorAssetPopover` component to `AssetInfoPopover`. It was initially built for HL7 Vitals Monitor and Ventilator Monitor Assets, but might have not been renamed make it more generic once we started using it for Camera Assets too.

```suggestion
                <AssetInfoPopover
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]
Nice work!

**Review by khavinshankar — APPROVED** [review]

**Comment by Rishith25** [comment]
@rithviknishad Can you please assign 2 story points for this

**Comment by rithviknishad** [comment]
@Rishith25 for minor changes we assign 1 story point.

**Pull request merged**